src/scales/quantitativeScale.ts

```diff
@@ -152,7 +153,28 @@ export class QuantitativeScale<D> extends Scale<D, number> {
     });
     var newMin = minExistsInExceptions ? min : this.invert(this.scale(min) - (this.scale(max) - this.scale(min)) * p);
     var newMax = maxExistsInExceptions ? max : this.invert(this.scale(max) + (this.scale(max) - this.scale(min)) * p);
-    return this._niceDomain([newMin, newMax]);
+    if (this._niceDomainEnabled) {
+      return this._niceDomain([newMin, newMax]);
+    }
+    return ([newMin, newMax]);
+  }
+
+  /**
+   * Gets whether or not the scale will have a nice domain.
+   */
+  public niceDomain(): boolean;
+  /**
+   * Sets whether or not the scale will have a nice domain.
+   */
+  public niceDomain(niceDomainEnabled: boolean): QuantitativeScale<D>;
```

Comment:
Generally our convention has been to have param name in a getter-setter carry the same name as the method itself, so `niceDomain` in this case.